tools/codegen/dest/native_functions.py

```diff
@@ -1,96 +1,59 @@
-from typing import List, Union, Set, Any
```

Comment:
One thing that's nice about this PR (although it's hard to tell from the git diff) is that this file got a lot shorter. `gen_unstructured()` and `gen_structured()` both do less since they only operate on a single (NativeFunction, backend) pair. And `gen_structured()` already knows that it's been handed a structured backend, so it doesn't need to handle multiple cases.